scripts/postinstall.js

```diff
@@ -1,31 +1,31 @@
+/* eslint no-console: "off" */
 const fs = require("fs");
-const allSets = require("../src/make/allsets");
 const configFiles = ["config.client.js", "config.server.js"];
-const logger = require("../src/logger");
+const allSets = require("../src/make/allsets");
 
 // Download Allsets.json
 if(!fs.existsSync("data/AllSets.json")) {
-  logger.info("No AllSets.json detected. Downloading the file...");
-  allSets.download("https://mtgjson.com/json/AllSets.json", "data/AllSets.json", () => {
-    logger.info("Download of AllSets.json completed");
+  console.log("No AllSets.json detected. Downloading the file...");
+  allSets.download("http://mtgjson.com/json/AllSets.json", "data/AllSets.json", () => {
```

Comment:
Did you remove https on purpose?

Reply:
Hi @tooomm ,

Yes it's deliberate, I encountered some errors. But after investigating, I can just modify the allsets.js that use the module "http" instead of "https"!